src/dredd.coffee

```diff
@@ -133,12 +152,12 @@ class Dredd
     , callback
 
   downloadFile: (fileUrl, callback) ->
-    request.get
-      url: fileUrl
-      timeout: 5000
-      json: false
-    , (downloadError, res, body) =>
+    options = {url: fileUrl, timeout: 5000, json: false}
```

Comment:
I somehow don't like the hardcoded timeout, I'd vote for extracting to the constant with some meaningful name at least, but I am not exactly sure about the use case, so it's up to you.